pippy/ManualPipelineStage.py

```diff
@@ -172,36 +176,55 @@ def get_stage_shapes(
 
 
 class ManualPipelineStage(PipelineStageBase):
+    """
+    A class representing a pipeline stage in a pipeline parallelism setup.
+    This class is created manually by providing a example input (and optionally output)
+    as opposed to the PipelineStage class that is outputed from pipeline().
+    This class extends the `PipelineStageBase` class and can similarly be used
+    in `PipelineScheule`.
+    Args:
+        submodule (nn.Module): The PyTorch module wrapped by this stage.
+        stage_index (int): The ID of this stage.
+        num_stages (int): The total number of stages.
+        device (torch.device): The device where this stage is located.
+        num_microbatches (int): The number of microbatches to use.
+        input_args (Union[torch.Tensor, List[torch.tensor]], optional): The input arguments for the module.
+        output_args (Union[torch.Tensor, List[torch.tensor]], optional): The output arguments for the module.
```

Comment:
> The input arguments for the module

What does "the module" refer to? 

Earlier, the doc uses a term "submodule":
> submodule (nn.Module): The PyTorch module wrapped by this stage.

Reply:
oops, meant "submodule," thanks!